core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -596,6 +599,13 @@ case class MisClassificationsPerCategory
   MisClassifications: Seq[ClassCount]
 )
 
+case class ConfusionMatrixPerThreshold
+(
+  Threshold: Double,
+  @JsonDeserialize(contentAs = classOf[java.lang.Long])
+  ConfusionMatrixCounts: Seq[Long]
```

Comment:
minor style issue: params are typically lower camel case:

```suggestion
  threshold: Double,
  @JsonDeserialize(contentAs = classOf[java.lang.Long])
  confusionMatrixCounts: Seq[Long]
```

Reply:
Looks like we would have to update to follow the camel case for other existing classes as well:
MisClassificationMetrics, MisClassificationsPerCategory, etc